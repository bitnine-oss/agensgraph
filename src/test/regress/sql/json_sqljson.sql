-- JSON_EXISTS

SELECT JSON_EXISTS(NULL FORMAT JSON, '$');

-- JSON_VALUE

SELECT JSON_VALUE(NULL FORMAT JSON, '$');

-- JSON_QUERY

SELECT JSON_QUERY(NULL FORMAT JSON, '$');

-- JSON_TABLE

SELECT * FROM JSON_TABLE(NULL FORMAT JSON, '$' COLUMNS (foo text));
