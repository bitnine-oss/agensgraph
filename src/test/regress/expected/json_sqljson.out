-- JSON_EXISTS
SELECT JSON_EXISTS(NULL FORMAT JSON, '$');
ERROR:  JSON_EXISTS() is not yet implemented for the json type
LINE 1: SELECT JSON_EXISTS(NULL FORMAT JSON, '$');
               ^
HINT:  Try casting the argument to jsonb
-- JSON_VALUE
SELECT JSON_VALUE(NULL FORMAT JSON, '$');
ERROR:  JSON_VALUE() is not yet implemented for the json type
LINE 1: SELECT JSON_VALUE(NULL FORMAT JSON, '$');
               ^
HINT:  Try casting the argument to jsonb
-- JSON_QUERY
SELECT JSON_QUERY(NULL FORMAT JSON, '$');
ERROR:  JSON_QUERY() is not yet implemented for the json type
LINE 1: SELECT JSON_QUERY(NULL FORMAT JSON, '$');
               ^
HINT:  Try casting the argument to jsonb
-- JSON_TABLE
SELECT * FROM JSON_TABLE(NULL FORMAT JSON, '$' COLUMNS (foo text));
ERROR:  JSON_TABLE() is not yet implemented for the json type
LINE 1: SELECT * FROM JSON_TABLE(NULL FORMAT JSON, '$' COLUMNS (foo ...
                                 ^
HINT:  Try casting the argument to jsonb
